docs/factored_gaussian.rst

```diff
@@ -0,0 +1,94 @@
+Factoring the Gaussian Filter
+=============================
+
+This section provides background on applying a Gaussian filter with a large filter scale by repeatedly applying Gaussian filters with a smaller filter scale.
+This was not discussed in `Grooms et al. (2021) <https://doi.org/10.1029/2021MS002552>`_, so this section provides extra detail.
+
+The :py:class:`gcm_filters.Filter` class has an argument ``n_iterations`` that will automatically split a Gaussian filter into a sequence of Gaussian filters with smaller scales, each of which is less sensitive to roundoff errors. If a user is encountering instability with the standard Gaussian filter, the user can try setting ``n_iterations`` to an integer greater than 1. In general ``n_iterations`` should be set to the smallest value that avoids numerical instability. The user should input their desired filter scale, and then the code will automatically adjust it to reflect the filter scale of the smaller constituent filters.
+Applying the Taper filter repeatedly is not equivalent to applying it once with a different scale, so this method only works for the Gaussian filter.
+
+.. note:: When ``n_iterations`` is greater than 1, ``n_steps`` is the number of steps in a single small-scale Gaussian filter. The total number of steps taken by the algorithm is the product of ``n_iterations`` and ``n_steps``. If ``n_steps`` is not set by the user, or if it is set to a value less than 3, the code automatically changes ``n_steps`` to a default value that gives an accurate approximation to the small-scale Gaussian filters that comprise the factored filter.
```

Comment:
It does not make sense to me that we would silently override an explicit value of `nsteps=3`. If this is not a valid choice, we should raise a `ValueError`. There should be one and only one way to trigger the automatic determination of `n_steps`, e.g. `n_steps=None`.

Reply:
This isn't really a comment about this PR per se; I'm just describing how the current code treats `n_steps`. Could we deal with this separately?